Makefile

```diff
@@ -2,7 +2,7 @@ MODULE_big	= pgnodemx
 OBJS		= pgnodemx.o cgroup.o envutils.o fileutils.o genutils.o kdapi.o parseutils.o
 PG_CPPFLAGS	= -I$(libpq_srcdir)
 EXTENSION	= pgnodemx
-DATA		= pgnodemx--1.0--1.1.sql pgnodemx--1.1.sql
+DATA		= pgnodemx--1.0--1.1.sql pgnodemx--1.1.sql pgnodemx--1.1--1.2.sql
```

Comment:
Per testing with Pg 10+ (might be earlier, but only tested w/10), this is no longer required; it appears that as long as there is a base that is reachable to the version given in the `.control` file it'll do the right thing:
```
[david@pgguru pgnodemx]$ psql -U postgres
psql (10.13)
Type "help" for help.

postgres=# create extension pgnodemx;
CREATE EXTENSION
postgres=# select * from pg_extension;
 extname  | extowner | extnamespace | extrelocatable | extversion | extconfig | extcondition
----------+----------+--------------+----------------+------------+-----------+--------------
 plpgsql  |       10 |           11 | f              | 1.0        |           |
 pgnodemx |       10 |         2200 | t              | 1.2        |           |
(2 rows)
```

Reply:
Thanks for checking that!  

Does make it challenging to know which update file has the current version of any object's SQL code, though.

Maybe can review the other ways to handle this for a future update then

Reply:
https://www.postgresql.org/docs/current/extend-extensions.html

> Before PostgreSQL 10, it was necessary to also create new script files `foo--1.1.sql` and `foo--1.2.sql` that directly build the newer extension versions, or else the newer versions could not be installed directly, only by installing `1.0` and then updating. That was tedious and duplicative, but now it's unnecessary, because `CREATE EXTENSION` can follow update chains automatically.